libensemble/gen_classes/aposmm.py

```diff
@@ -27,52 +36,122 @@ class APOSMM(PersistentGenInterfacer):
         "x_on_cube": ["var1_on_cube", "var2_on_cube", "var3_on_cube"],
     }
     gen = APOSMM(vocs, variables_mapping=variables_mapping, ...)
+
+    Parameters
+    ----------
+    vocs: VOCS
+        The VOCS object, adhering to the VOCS interface from the Generator Standard.
+
+    History: npt.NDArray = []
+        An optional history of previously evaluated points.
+
+    initial_sample_size: int = 100
```

Comment:
Should be dependent on `n` or possibly `nfmax`

Reply:
I'm making it a required field as we discussed - I'm assuming users will eyeball a good stating size?